src/App.jsx

```diff
@@ -1,26 +1,64 @@
-/* eslint-disable */
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import calculate from './logic/calculate';
 import './App.css';
 
-function App() {
+class Calculator extends Component {
```

Comment:
- [ ] kindly recreate the App component and import the Calculator here instead of using the same component code inside the app.jsx, (your using the code of Calculator.jsx in your App.jsx file)